pkg/cmd/server/logs.go

```diff
@@ -120,7 +148,57 @@ func readServerLogFile() error {
 	}
 }
 
-func init() {
-	logsCmd.Flags().BoolVarP(&followFlag, "follow", "f", false, "Follow logs")
-	logsCmd.Flags().BoolVar(&fileFlag, "file", false, "Read logs from local server log file")
+func getLogFiles(dir string) ([]string, error) {
+	files, err := os.ReadDir(dir)
+	if err != nil {
+		return nil, err
+	}
+
+	var logFiles []string
+	for _, file := range files {
+		if file.Name() == "daytona.log" || strings.HasPrefix(file.Name(), "daytona-") && (strings.HasSuffix(file.Name(), ".log") || strings.HasSuffix(file.Name(), ".zip")) {
```

Comment:
`.tar.gz` is also a valid extension. I had log files with `.tar.gz` when testing the log rotation.